habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -475,13 +465,6 @@ def sim_update(self, dt):
             for keyframe in keyframes:
                 self._recording_keyframes.append(keyframe)
 
-        # Manually save recorded gfx-replay keyframes.
```

Comment:
This looks like debug code. Removing it. It can be re-added to individual AppStates that need it.